e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/jsonarray/WmJsonArrayEMTest.kt

```diff
@@ -34,7 +34,7 @@ class WmJsonArrayEMTest : SpringTestBase() {
             "WmJsonArrayEM",
             "org.foo.WmJsonArrayEM",
             1000,
-            true,
+            false,
```

Comment:
Had to disable it, because the generated tests has inconsistencies. Which are similar to existing issues under `issue-wm-e2e`.

Reply:
if it is temporary, must add a `/TODO` comment explaining it

Reply:
riveted back to original, now passing